collections/_posts/2022-10-03-MomentumResidualNeuralNetwork.md

```diff
@@ -0,0 +1,151 @@
+---
+layout: review
+title: "Momentum Residual Neural Networks"
+tags: Invertible ResNets
+author: "Stéphanie Jehan-Besson"
+cite:
+    authors: "Michael E. Sander, Pierre Ablin, Mathieu Blondel, Gabriel Peyré"
+    title:   "Momentum Residual Neural Networks"
+    venue:   "ICML 2021"
+pdf: "https://arxiv.org/pdf/2102.07870.pdf" 
+---
+
+<!-- Do not include the following disclaimer in your own review -->
+
+# Reminders on Residual Neural Networks
+
+Seminal work on Residual Neural Network : ["Deep Residual Learning for Image Recognition", K. He, X. Zhang, R. Zen and J. Sun, CVPR 2016](https://arxiv.org/pdf/1512.03385.pdf)
+
+Main points:
+
+* Deeper neural networks are more difficult to train
+* Residual learning framework have been introduced to ease the training of deeper networks
+* Layers are reformulated as learning residual functions with reference to the previous layer inputs, instead of learning unreferenced functions.
+* These residual networks are easier to optimize, and can gain accuracy from considerably increased depth.
+
+
+# Highlights of the paper
+
+* Exploration of Memory costs issues in residual neural network due to backpropagation schemes
+* Proposition of a new model, Momentum ResNets, which circumvents these memory issues by being invertible
+* Compared to other invertible models, this model is simple to integrate into the usual ResNets architectures and provides a rigorous mathematical setting.
+* Momentum ResNets can be interpreted as second order ordinary differential equations (ODEs).
+Momentum ResNets separate point clouds that ResNets fail to separate
+
+## Bibliography in the field 
+* Momentum in deep networks : (He et al., 2020; Chun et al., 2020; Nguyen et al., 2020; Li et al. 2018) 
+* Second-order models : (Norcliffe et al., 2020; Rusch & Mishra, 2021; Lu et al.,2018; Massaroli et al., 2020) 
+
+# Introduction
+
+The main goal of this paper is to explore the properties of a new model, Momentum ResNets, that circumvent the memory issues of ResNets by being invertible. It lies on the modification of the ResNet's forward rule.
+
+# Momentum Residual Neural Networks
+
+Backpropagation, used to optimize deep architectures, requires to store values at each layer during the network training.
+In classical ResNets, we have the feedforward relation:
+
+$$x_{n+1}=x_n+ f(x_n,\theta_n)$$
+
+where $$\theta_n$$ is the set of parameters, and $$x_n$$ the activations at layer $$n$$. 
+
+Memory issues occur when increasing the number of layers.
+
+The authors propose to use momentum equations that replace the classical relation above:
+
+$$ v_{n+1}=\gamma v_n+(1-\gamma)f(x_n,\theta_n)$$
```

Comment:
It can be interesting to say how v_0 is chosen and if the chosen value has an impact on the final result ?
I just have seen that you say something on this point in the result section ;) Maybe just mention this here !

Reply:
I will do it for the final version